src/Exception/TestException.php

```diff
@@ -10,7 +10,7 @@
 /**
  * 
  */
-abstract class ExceptionAbstract extends \Exception
+abstract class TestException extends \Exception
```

Comment:
Let's not worry about this too much. You could also define interfaces for exceptions. Let's just rename this to AbstractTestException and keep the TestNotFoundException as well